package.json

```diff
@@ -9,6 +9,7 @@
     }
   ],
   "scripts": {
+    "start": "gulp dev",
```

Comment:
This does not appear to work when `gulp-cli` is not installed globally or `gulp` is otherwise added as a PATH environment var or linked to

Making `gulp-cli` a dev dependency doesn't seem to make any difference either.